src/components/organisms/Navbar/NavbarLink/NavbarLink.tsx

```diff
@@ -1,53 +1,99 @@
 import React, { FC } from 'react';
-import styled from 'styled-components';
-import { colors } from '../../../../constants';
+import styled, { css } from 'styled-components';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+
+import { colors, spacing, typography } from '../../../../constants';
+import { minMedia, maxMedia } from '../../../../helpers/responsiveness';
+
 import Link, { LinkProps } from '../../../atoms/Link/Link';
 import Icon from '../../../atoms/Icon/Icon';
-import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+
+export interface NavbarLinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  active?: boolean;
+  withChevron?: boolean;
+  isDropdownLink?: boolean;
+  open?: boolean;
+  target?: LinkProps['target'];
+  'data-automation'?: string;
+}
 
 export interface StyledNavbarLinkProps extends LinkProps {
   active: boolean;
   withChevron: boolean;
+  isDropdownLink: boolean;
 }
 
-const StyledNavbarLink = styled(Link)<StyledNavbarLinkProps>`
-  display: inline-flex;
+export interface ChevronContainerProps extends React.HTMLAttributes<HTMLSpanElement> {
+  open: boolean;
+}
+
+export const NavbarLinkStyles = css<StyledNavbarLinkProps>`
   align-items: center;
```

Comment:
we used camel case for `linkStyles` and `buttonStyles` - would be nice to be consistent here too